rmk/src/lib.rs

```diff
@@ -195,18 +197,29 @@ pub async fn run_rmk<
     light_controller: &mut LightController<Out>,
     rmk_config: RmkConfig<'static>,
 ) -> ! {
+    #[cfg(feature = "controller")]
```

Comment:
Is this the right place to initialize the controller?

Reply:
I think the controller initialization should be controlled by users's `main.rs`, just like input devices. For toml config users, the controller initialization code is generated by rmk-macro according to the config. It also makes it possible for users to write their own controllers.

The drawbacks is that it makes the user-side code quite complex. I'm not sure whether this way is clear enough, what do you think about it?

Reply:
I guess we can leave this out for now, I'll drop this commit.